src/consumer.ts

```diff
@@ -362,8 +362,9 @@ export class Consumer extends TypedEventEmitter {
     } catch (err) {
       this.emitError(err, message);
 
-      if (this.terminateVisibilityTimeout) {
-        await this.changeVisibilityTimeout(message, 0);
+      if (this.terminateVisibilityTimeout !== false) {
+        const timeout = this.terminateVisibilityTimeout === true ? 0 : this.terminateVisibilityTimeout
```

Comment:
Thanks for the PR, this makes sense. I am currently thinking if it makes sense to add a new option though (`terminateVisibilityTimeoutMs`) rather than this logic and then just default that to 0 and allow it to be changed.

Reply:
Thanks for the review! To be honest I'm on the fence too. If we reuse the `terminateVisibilityTimeout` option, the naming is a bit confusing; if we add a `terminateVisibilityTimeoutMs`, without looking at their types, there will appear to be a timeout and a timeout in milliseconds at the same time, and I'm not sure what should happen if `terminateVisibilityTimeout == false` but `terminateVisibilityTimeoutMs` is set.

I'm happy to leave the decision to you. I can update the code and docs if you opt for a new option in the end.

PS: Forgot to `npm run format`. Just pushed :)

Reply:
Yeah that makes sense too, I think I'll just merge this into canary for now and think on it, if I don't decide against it by Wednesday, I'll push it up to main.

Thanks a lot for the contribution!